website/developer-docs/setup/full-dev-environment.md

```diff
@@ -18,15 +18,19 @@ You can also use a native install, if you prefer.
 
 ## Backend Setup
 
+:::info
+Depending on platform, some native dependencies might be required. On macOS, run `brew install libxmlsec1 libpq`, and for the CLI tools `brew install postgresql redis node@20`
```

Comment:
```suggestion
Depending on your platform, some native dependencies might be required. On MacOS, run `brew install libxmlsec1 libpq`, and for the CLI tools `brew install postgresql redis node@20`
```

Reply:
macOS is actually stylized lowercase nowadays https://en.wikipedia.org/wiki/MacOS

```suggestion
Depending on your platform, some native dependencies might be required. On macOS, run `brew install libxmlsec1 libpq`, and for the CLI tools `brew install postgresql redis node@20`
```

Reply:
Eventually we need to decide either how to handle writing with OS-agnosticism, or... provide a tabbed .mdx section with commands for each OS.